task-list/app/assets/stylesheets/tasks.scss

```diff
@@ -75,10 +75,18 @@ footer  {
 
 }
 
+.comp_date {
+  font-weight: bold;
+  font-size: 15px;
+  
+}
 
+#task_person_id {
```

Comment:
CSS styles that contain multiple words should be separated by the `-` not the `_`